chain/client/src/client_actor.rs

```diff
@@ -161,6 +176,9 @@ impl ClientActor {
             doomslug_timer_next_attempt: now,
             chunk_request_retry_next_attempt: now,
             sync_started: false,
+            state_parts_task_scheduler: Box::new(move |msg: ApplyStatePartsRequest| {
+                state_parts_actor_addr.do_send(msg);
+            }),
```

Comment:
What is the advantage of doing this versus having state_parts_actor_addr directly?

Reply:
Among other things quite a few tests call some intermediate methods without benefit of having ClientActor